### PR TITLE
Navigation added to all relevant pages

### PR DESCRIPTION
**Pull request by shaybrow**

### CODE REVIEW

File: app/src/main/java/com/trueproof/trueproof/activities/BatchDetailActivity.java [hunk 1]
```diff
@@ -58,9 +64,24 @@ private void getBatchFromIntent(Intent intent) {
...
+    public boolean onCreateOptionsMenu (Menu menu){
+        getMenuInflater().inflate(R.menu.menu, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem menuItem){
+        if (menuItem.getItemId() == R.id.nav_settings)BatchDetailActivity.this.startActivity(new Intent(BatchDetailActivity.this, SettingsActivity.class));
+        if (menuItem.getItemId() == R.id.nav_batch_list)BatchDetailActivity.this.startActivity(new Intent(BatchDetailActivity.this, BatchListActivity.class));
+        if (menuItem.getItemId() == R.id.nav_quick_calculator)BatchDetailActivity.this.startActivity(new Intent(BatchDetailActivity.this, TakeMeasurementActivity.class));
```
Comment (wordhou) [inline app/src/main/java/com/trueproof/trueproof/activities/BatchDetailActivity.java]: Make this direct to MainActivity.class?

File: app/src/main/java/com/trueproof/trueproof/activities/BatchListActivity.java [hunk 2]
```diff
@@ -124,4 +126,17 @@ private void onActivityResult(ActivityResult activityResult) {
...
+    public boolean onCreateOptionsMenu (Menu menu){
+        getMenuInflater().inflate(R.menu.menu, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem menuItem){
+        if (menuItem.getItemId() == R.id.nav_settings)BatchListActivity.this.startActivity(new Intent(BatchListActivity.this, SettingsActivity.class));
+        if (menuItem.getItemId() == R.id.nav_batch_list)BatchListActivity.this.startActivity(new Intent(BatchListActivity.this, BatchListActivity.class));
+        if (menuItem.getItemId() == R.id.nav_quick_calculator)BatchListActivity.this.startActivity(new Intent(BatchListActivity.this, TakeMeasurementActivity.class));
```
Comment (wordhou) [inline app/src/main/java/com/trueproof/trueproof/activities/BatchListActivity.java]: Etc.

File: app/src/main/java/com/trueproof/trueproof/activities/MeasurementDetailActivity.java [hunk 3]
```diff
@@ -13,4 +16,18 @@ protected void onCreate(Bundle savedInstanceState) {
...
+    public boolean onCreateOptionsMenu (Menu menu){
+        getMenuInflater().inflate(R.menu.menu, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem menuItem){
+        if (menuItem.getItemId() == R.id.nav_settings)MeasurementDetailActivity.this.startActivity(new Intent(MeasurementDetailActivity.this, SettingsActivity.class));
+        if (menuItem.getItemId() == R.id.nav_batch_list)MeasurementDetailActivity.this.startActivity(new Intent(MeasurementDetailActivity.this, BatchListActivity.class));
+        if (menuItem.getItemId() == R.id.nav_quick_calculator)MeasurementDetailActivity.this.startActivity(new Intent(MeasurementDetailActivity.this, TakeMeasurementActivity.class));
```
Comment (wordhou) [inline app/src/main/java/com/trueproof/trueproof/activities/MeasurementDetailActivity.java]: Etc.

File: app/src/main/java/com/trueproof/trueproof/activities/NewBatchActivity.java [hunk 4]
```diff
@@ -112,6 +114,18 @@ private void batchIdentifierOnChange (){
...
+    public boolean onCreateOptionsMenu (Menu menu){
+        getMenuInflater().inflate(R.menu.menu, menu);
+        return true;
+    }
 
+    @Override
+    public boolean onOptionsItemSelected(MenuItem menuItem){
+        if (menuItem.getItemId() == R.id.nav_settings)NewBatchActivity.this.startActivity(new Intent(NewBatchActivity.this, SettingsActivity.class));
+        if (menuItem.getItemId() == R.id.nav_batch_list)NewBatchActivity.this.startActivity(new Intent(NewBatchActivity.this, BatchListActivity.class));
+        if (menuItem.getItemId() == R.id.nav_quick_calculator)NewBatchActivity.this.startActivity(new Intent(NewBatchActivity.this, TakeMeasurementActivity.class));
```
Comment (wordhou) [inline app/src/main/java/com/trueproof/trueproof/activities/NewBatchActivity.java]: Etc.

File: app/src/main/java/com/trueproof/trueproof/activities/SettingsActivity.java [hunk 5]
```diff
@@ -125,4 +128,17 @@ public void onRadioButtonUserSettings (View view){
...
+    public boolean onCreateOptionsMenu (Menu menu){
+        getMenuInflater().inflate(R.menu.menu, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem menuItem){
+        if (menuItem.getItemId() == R.id.nav_settings)SettingsActivity.this.startActivity(new Intent(SettingsActivity.this, SettingsActivity.class));
+        if (menuItem.getItemId() == R.id.nav_batch_list)SettingsActivity.this.startActivity(new Intent(SettingsActivity.this, BatchListActivity.class));
+        if (menuItem.getItemId() == R.id.nav_quick_calculator)SettingsActivity.this.startActivity(new Intent(SettingsActivity.this, TakeMeasurementActivity.class));
```
Comment (wordhou) [inline app/src/main/java/com/trueproof/trueproof/activities/SettingsActivity.java]: Etc.

File: app/src/main/java/com/trueproof/trueproof/activities/TakeMeasurementActivity.java [hunk 6]
```diff
@@ -244,6 +246,18 @@ public String userLocalTime(){
...
+    public boolean onCreateOptionsMenu (Menu menu){
+        getMenuInflater().inflate(R.menu.menu, menu);
+        return true;
+    }
 
+    @Override
+    public boolean onOptionsItemSelected(MenuItem menuItem){
+        if (menuItem.getItemId() == R.id.nav_settings)TakeMeasurementActivity.this.startActivity(new Intent(TakeMeasurementActivity.this, SettingsActivity.class));
+        if (menuItem.getItemId() == R.id.nav_batch_list)TakeMeasurementActivity.this.startActivity(new Intent(TakeMeasurementActivity.this, BatchListActivity.class));
+        if (menuItem.getItemId() == R.id.nav_quick_calculator)TakeMeasurementActivity.this.startActivity(new Intent(TakeMeasurementActivity.this, TakeMeasurementActivity.class));
```
Comment (wordhou) [inline app/src/main/java/com/trueproof/trueproof/activities/TakeMeasurementActivity.java]: Etc.

**Review by wordhou — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/trueproof/trueproof/activities/BatchDetailActivity.java (hunk 1 above)
Comment (shaybrow) [reply app/src/main/java/com/trueproof/trueproof/activities/BatchDetailActivity.java]: done on all counts

**Review by wordhou — APPROVED** [review]

**Pull request merged**